pyiron_nodes/atomistic/calculator/data.py

```diff
@@ -1,9 +1,10 @@
-from __future__ import annotations
+# from __future__ import annotations
```

Comment:
I had to remove this line to get @as_dataclass_node to work properly. The inclusion of this future statement caused type hints in the node to be strings instead of actual types. It took me quite a while to figure out this strange behavior. Since most of us are working with a Python version well above 3.9, dropping support for 3.9 should be fine. So I strongly support your suggestion to drop it everywhere.